api-client/src/utils/licences.ts

```diff
@@ -0,0 +1,25 @@
+export type JsonLicenses = {
+  [name: string]: {
+    licenses: string
+    licenseUrl: string
+  }
+}
+
+export type License = {
+  name: string
+  version: string | undefined
+  licenses: string
+  licenseUrl: string
+}
+
+const numberRegex = /\d+(\.\d+)*/
+const versionRegex = /(?:@)\d+(\.\d+)*/
+
+export const parseLicenses = (licenseFile: JsonLicenses): License[] =>
+  Object.entries(licenseFile).map(([name, { licenses, licenseUrl }]) => {
+    // Extract the version of the library from the name
+    // get license name without version in it
```

Comment:
Comments are a little bit unnecessary imo
```suggestion
```

Reply:
Not at all. The integreat code is hard to undestand because it barely has any comments